scripts/verify-phase-37-runtime-smoke-gate.sh

```
@@ -0,0 +1,116 @@
+required_gate_phrases=(
+  'Usage: scripts/run-phase-37-runtime-smoke-gate.sh --env-file <runtime-env-file> [--evidence-dir <evidence-dir>]'
+  'AEGISOPS_SMOKE_PLATFORM_ADMIN_SUBJECT'
+  'AEGISOPS_SMOKE_PLATFORM_ADMIN_IDENTITY'
+  'AEGISOPS_SMOKE_READONLY_SUBJECT'
+  'AEGISOPS_SMOKE_READONLY_IDENTITY'
+  'AEGISOPS_SMOKE_READONLY_ROLE'
+  'AEGISOPS_SMOKE_REVIEWED_ACTION_SCOPE_ID'
+  'AEGISOPS_SMOKE_LOW_RISK_ACTION_TYPE'
+  'AEGISOPS_SMOKE_APPROVER_OWNER'
+  'Invalid smoke low-risk action type: AEGISOPS_SMOKE_LOW_RISK_ACTION_TYPE must be notify_identity_owner'
+  'bash "${repo_root}/scripts/verify-customer-like-rehearsal-environment.sh" --env-file "${env_file}"'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)